### PR TITLE
Remove call to non-existent 'ensime-sbt-do-scalariform-only'

### PR DESCRIPTION
**Pull request by mmajcher**
'ensime-sbt-do-scalariform-only' has been removed by:
3d3ab18436ad6089496b3bce1d49c64a86965431

### CODE REVIEW

File: ensime-editor.el [hunk 1]
```diff
@@ -706,8 +706,7 @@ If user selects an import, add it to the import list."
 Functionality was moved from ensime-server to the build tool plugins.
 Use build tools tasks appropriately"
   (interactive)
-  (message "Function use is deprecated, please transition to ensime-sbt-do-scalariform-only.")
-  (ensime-sbt-do-scalariform-only))
+  (message "Function use is deprecated, please transition to ensime-sbt-do-scalariform-only."))
```
Comment (iravid) [inline ensime-editor.el]: Would it make sense to get rid of `ensime-format-source` entirely?

**Comment by mmajcher** [comment]
Removed all of 'scalariform' references

**Comment by iravid** [comment]
Cool. Any objections? cc @fommil @yyadavalli

**Comment by fommil** [comment]
cool. Generally in 3.0 I'd like us to move towards a small kernel that understands the `.ensime` file and server protocol, and have a separate "contrib" module for anything like this.

I've also asked the sbt-scalariform maintainers to take ownership of the `scalariformOnly` task that we used to have, but they don't seem to want it. I don't care because I've moved onto scalafmt for everything.

**Pull request merged**